common/lib/container-definitions/src/loader.ts

```diff
@@ -251,7 +264,7 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
     /**
      * Provides the current connected state of the container
      */
-    readonly connectionState: ConnectionState;
+    readonly connectionState: Exclude<ConnectionState, ConnectionState.TryingToConnect>;
```

Comment:
Back compat. If someone were checking this state against Disconnected state, and suddenly Disconnected is divided into two states, they would break.

Reply:
In the following major release I'll cut it over, after giving fair warning. Or even deprecate this one and add a new one with the new type to be even more explicit. 

In Loop code the property is unused, so given that data point it may be more of a formality.

Reply:
I think you are talking about runtime behavior, not about type. Type can be anything, and having new cases allows customers to start writing new code to be prepared for new values. Adding them in runtime should be delayed and happen sometime after new types are added.

Reply:
So you're suggesting that we declare this as the breaking moment, where at any point in 1.0 timeframe the implementation may start dividing Disconnected into Disconnected and TryingToConnect?

Reply:
The most explicit alternative is to keep the Exclude but mark this as deprecated, and add a new property `connectionState2` which includes all the values.  Then over the next few releases delete this old one and rename the new one to drop the `2`.

Based on the limited (none known) usage of this property I will go ahead and just break it all at once, and depend on consumers reading BREAKING.md to understand that Disconnected state is being divided in two.